linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_pwhistory_remember/ansible/shared.yml

```diff
@@ -3,18 +3,18 @@
 # strategy = configure
 # complexity = low
 # disruption = medium
-- (xccdf-var var_password_pam_unix_remember)
```

Comment:
This is a backwards incompatible change and will break many peoples tailoring files. Please don't do this.